DEVELOPERS.md

```diff
@@ -5,7 +5,7 @@ Information for Code Climate CLI developers and contributors.
 Build a new image using the local sources:
 
 ```console
-docker build --rm -t codeclimate/codeclimate .
+make image
```

Comment:
I don't think this is necessary, [bin/release](https://github.com/codeclimate/codeclimate/blob/master/bin/release#L20) builds the image